apps/web/pages/api/v1/teams/[teamId]/invite/[inviteId]/index.ts

```diff
@@ -112,6 +113,20 @@ export default async function handle(req: NextApiRequest, res: NextApiResponse)
 
     return res.status(200).json(updatedInvite);
   }
+  // GET /api/v1/teams/[teamId]/invite/[inviteId]?email=[email]
+  // Retrieve an invite token
+  else if (req.method === "GET") {
```

Comment:
I think it would make sense to check if an invite exists for this email address - just to be on the safe side 😊.

Reply:
Just thought about it again. You don't need the email address as a query parameter here, you need to pull the invite from the database to get the email address from it 😊.

Reply:
thanks for the feedback, will work on it.

Reply:
just committed new adjusted as you suggested.